docs/privacy_policy.md

```diff
@@ -0,0 +1,54 @@
+## Privacy Policy
+
+Georepublic UG (haftungsbeschränkt) built the Consento app as an Open Source app. This SERVICE is provided by Georepublic UG (haftungsbeschränkt) at no cost and is intended for use as is.
+
+This page is used to inform visitors regarding our policies with the collection, use, and disclosure of Personal Information if anyone decided to use our Service.
+
+If you choose to use our Service, then you agree to the collection and use of information in relation to this policy. The Personal Information that we collect is used for providing and improving the Service. We will not use or share your information with anyone except as described in this Privacy Policy.
+
+The terms used in this Privacy Policy have the same meanings as in our Terms and Conditions, which is accessible at Consento unless otherwise defined in this Privacy Policy.
```

Comment:
What "terms & conditions"?

Reply:
Can "Terms and Conditions" be linked to that page ( if there are any? ) Or is that not good formatting for a privacy policy?